contracts/interfaces/INXMMaster.sol

```diff
@@ -22,11 +20,7 @@ interface INXMMaster {
 
   function checkIsAuthToGoverned(address _add) external view returns (bool);
 
-  function updatePauseTime(uint _time) external;
-
   function dAppLocker() external view returns (address _add);
```

Comment:
Ideally we should remove both ` dAppLocker` and `dAppToken` but we'd need to make sure that no contract (including external contracts) are calling it.

Reply:
governance is currently using those..